### ISSUE TITLE
Kustom API have been updated to 2.0

### ISSUE THREAD

**Issue by frmz** [jahirfiquitiva/Kuper#78]
### Kuper Version

2.4.4

### Describe the feature

Hello! I see you have forked kustomapi, shouldn't bee needed anymore, took the time to upgrade API libs and deps to a more recent release. You can migrate to kustom api 2.0:
https://central.sonatype.com/artifact/org.bitbucket.frankmonza/kustomapi/20

If you find an issue just let me know by opening an incident here:
https://bitbucket.org/frankmonza/kustomapi/issues?status=new&status=open

Thanks!

**Comment by jahirfiquitiva** [comment]
interesting @frmz ... thanks for letting me know ... Are there any breaking/critical changes in the update? 😅

**Comment by frmz** [comment]
> interesting @frmz ... thanks for letting me know ... Are there any breaking/critical changes in the update? 😅

Not really, just bumped gradle version, removed legacy android support library, removed Async Task in preset library (in case you use it), minSDK is now 23, everything else is the same

**Comment by jahirfiquitiva** [comment]
sounds good @frmz .. will look into this soon, thanks! 🙌

**Comment by jahirfiquitiva** [comment]
@frmz I just realized Kuper uses minSDK 21 ... any chance of setting the minSDK in your library to 21 too?

**Comment by jahirfiquitiva** [comment]
@frmz also, the latest version is not `2.0` but `20`... you probably missed the dot 😅

**Comment by frmz** [comment]
> @frmz also, the latest version is not `2.0` but `20`... you probably missed the dot 😅

I indeed missed the dot but i do not want to go back as it would break some IDE that suggests upgrading, so i will just using integer releases from now on, not too bad. I can not set minSDK to 21 as Kustom now requires version 26, basically no one is in API < 26 anyway (its like less than 1%)

**Comment by jahirfiquitiva** [comment]
@frmz 

Have the intents for the Kustom apps changed? If so, which ones are the newest ones?

These are the ones I'm currently using, but I think they're not working as expected
- org.kustom.widget/org.kustom.widget.picker.WidgetPicker
- org.kustom.wallpaper/org.kustom.lib.editor.WpAdvancedEditorActivity
- org.kustom.lockscreen/org.kustom.lib.editor.LockAdvancedEditorActivity


I'm also facing errors like 
```gradle
java.lang.SecurityException: Permission Denial: starting Intent { dat=kfile://dev.jahir.kuper.app.kustom.provider/... cmp=org.kustom.lockscreen/org.kustom.lib.editor.LockAdvancedEditorActivity } from ProcessRecord{2cedf7b 17119:dev.jahir.kuper.app/u0a249} (pid=17119, uid=10249) not exported from uid 10273
```

This is the intent code
```kotlin
val intent = Intent().apply { setComponent(component) }
try {
    intent.data = Uri.Builder()
        .scheme("kfile")
        .authority("${context.packageName}.kustom.provider")
        .appendPath(path)
        .build()
} catch (e: Exception) {
    intent.data = Uri.parse("kfile://${context.packageName}/$path")
}
return intent
```

**Comment by jahirfiquitiva** [comment]
@frmz The queries are included. Anyway, opening a widget from Kuper (with the intent previously shared) does not open the widget editor correctly

**Comment by jahirfiquitiva** [comment]
@frmz I have uploaded a debug and a release apk here https://box.jahir.dev/shared/bbb4d143e1c2c58c

Please let me know how the tests go. Thanks!

**Comment by jahirfiquitiva** [comment]
@frmz Do you have any updates on this?

**Comment by frmz** [comment]
This is now fixed in Kustom version >= 3.74b328516 (its available via manual download now, being approved in the play store)
